### PR TITLE
Updated example config to use recommended settings instead of defaul,…

### PR DESCRIPTION
**Pull request by sdorheim**
… rewrote a bit and added sections to the Readme

### CODE REVIEW

**Review by spex66 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Purpose:
 - [InSo Bootstrap CLI](#inso-bootstrap-cli)
```
[the remark this answers is not in the document]
Comment (sdorheim) [reply README.md:22]: @spex66, is there a tool to update or just manual procedure?

File: README.md [hunk 2]
```diff
@@ -53,6 +54,121 @@ Purpose:
   - [run local with Docker](#run-local-with-docker)
   - [run as github action](#run-as-github-action)
 
+
+## How to get started
+
+The recommended way to run this is using poetry, but other methods are supported.
+For more details on other methods or native windows usage, check out [How to run](#how-to-run).
+To start you have to install Poetry, a tool to manage python dependencies and virtual environments. It is recommended running this on Linux, WSL2 or Mac.
```
[the remark this answers is not in the document]
Comment (sdorheim) [reply README.md]: I took it from our internal Python-setup guide, but I agree, will change

File: README.md [hunk 3]
```diff
@@ -137,180 +253,257 @@ Good style is to keep the names short and add long names and details to the `des
      3. allowing data-lineage from sources through use-case model to data-products
 
 ### Bootstrap CLI example
-An example config with the main abilities of the CLI. Shared owner access is also possible, but omitted here for simplicity.
+Here is an extract from the example config `config-simple-v2-draft.yml` wich uses the main abilities of the CLI. 
+
```
[the remark this answers is not in the document]
Comment (sdorheim) [reply README.md:286]: This has been fixed now

File: .env_example [hunk 4]
```diff
@@ -1,7 +1,6 @@
-BOOTSTRAP_CDF_HOST=**
+BOOTSTRAP_CDF_HOST=https://<Cluster Id>.cognitedata.com
```
[the remark this answers is not in the document]
Comment (sdorheim) [reply .env_example:1]: This has been updated

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sdorheim) [reply README.md:22]: Done

File: README.md [hunk 5]
```diff
@@ -53,6 +54,121 @@ Purpose:
...
+            BOOTSTRAP_IDP_CLIENT_ID: ${{ secrets.CLIENT_ID }}
+            BOOTSTRAP_IDP_CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+            BOOTSTRAP_CDF_HOST: ${{ env.CDF_HOST }}
+            BOOTSTRAP_CDF_PROJECT: ${{ env.CDF_PROJECT }}
+            BOOTSTRAP_IDP_TOKEN_URL: https://login.microsoftonline.com/${{ env.IDP_TENANT }}/oauth2/v2.0/token
+            BOOTSTRAP_IDP_SCOPES: ${{ env.CDF_HOST }}.default
+        # additional parameters for running the action
+        with:
+          config_file: ./configs/config-simple-v2-draft.yml
+          # "yes"|"no" deploy with special groups and aad_mappings
```
[the remark this answers is not in the document]
Comment (sdorheim) [reply README.md]: Done

File: README.md [hunk 6]
```diff
@@ -736,34 +946,3 @@ Debug the Docker container
 ➟  docker run --volume ${PWD}/configs:/configs --volume ${PWD}/logs:/logs  --env-file=.env -it --entrypoint /bin/bash incubator/bootstrap-cli:debug
 ```
```
[the remark this answers is not in the document]
Comment (sdorheim) [reply README.md]: Done

File: README.md [hunk 7]
```diff
@@ -53,6 +54,121 @@ Purpose:
...
+
+
+### Running locally
+
+With To set create a group with the proper access-rights for the bootstrap-cli to do it's job you can run the `prepare`-command. this creates a group and links it to a group the app-registration is in. 
+
+PS. It is possible to run all of commands in dry-run mode by specifying `--dry-run=yes` before the command. This wil log the intended API-actions.
+
+```
+poetry run bootstrap-cli --debug prepare --aad-source-id <idb-source-id>
```
[the remark this answers is not in the document]
Comment (sdorheim) [reply README.md]: done

**Review by spex66 — APPROVED** [review]
thx for the effort
some final changes (cleanup and renamed `configs/` examples)

**Pull request merged**